src/main/webapp/script.js

```diff
@@ -61,9 +69,16 @@ function fetchList(queryString) {
       totalCardCount++;
     }); 
     initialDisplay();
-    map.fitBounds(bounds);
+    map.fitBounds(bounds);  
+    removeLoader();
   });
 }
+ 
+function removeLoader() {
+  document.getElementById('loader').style.display = 'none';
+  document.getElementById('loader-container').style.display = 'none';
+  document.getElementById('map').style.opacity = '1';
```

Comment:
Can you add appropriate classes to the css file, and assign those here instead? Same as above.

Reply:
Just created the classes, let me know if the change makes sense.